release-calendar/package.json

```diff
@@ -10,30 +10,44 @@
     "url": "https://github.com/ampproject/amp-github-apps.git"
   },
   "engines": {
-    "node": "^12.13.1"
+    "node": "12.13.1"
   },
   "scripts": {
     "build": "tsc",
-    "dotenv": "node -r dotenv/config",
-    "lint": "npm run build && eslint **/*.ts --quiet --fix",
+    "lint": "eslint --quiet --fix src/* test/* --ext .ts,.tsx ",
     "start": "npm run build && node -r dotenv/config ./dist/app.js",
-    "test": "jest --verbose=false"
+    "test": "jest --verbose=false",
+    "client": "npm run client-build && npm run client-serve",
+    "client-build": "webpack --config ./webpack.client.config.js",
+    "client-serve": "webpack-dev-server --mode development --open --hot --config ./webpack.client.config.js",
   },
   "dependencies": {
-    "dotenv": "8.2.0",
```

Comment:
why did you remove this?

Reply:
I moved it below under `devDependencies`

Reply:
oh, duh, haha

Reply:
@estherkim _is it_ a dev dependencies? We use `dotenv` as a code dependency in other projects so I'm just wondering if this is not the case here either... what is it used for?

Reply:
It's to replace `process.env` variables. One way is to run it at app start like `require('dotenv').config()` so maaaaybe you need it in dependencies?

But I used the CLI way and put it in a node script like `node -r dotenv/config ./dist/app.js`. So as long as the variables get replaced before `dist/` is deployed then I think it's okay. I thiiiink